sky/serve/replica_managers.py

```diff
@@ -63,7 +65,14 @@ def launch_cluster(task_yaml_path: str,
             if retry.
     """
     try:
-        task = sky.Task.from_yaml(task_yaml_path)
+        with open(os.path.expanduser(task_yaml_path), 'r',
+                  encoding='utf-8') as f:
+            config = yaml.safe_load(f)
```

Comment:
nit: copy-paste the TODO here? https://github.com/skypilot-org/skypilot/blob/e93d400cb09acd111eb6f6c13854692b901c87d0/sky/task.py#L464-L466